CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -245,6 +269,32 @@ function Assert-IsAgentServiceDisabled {
     throw "Agent service is not disabled. EXPECTED: Agent service is disabled"
 }
 
+function Test-IsAgentServiceRegistered {
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session)
+
+    $Service = Invoke-Command -Session $Session -ScriptBlock {
+        return $(if (Get-Service "ContrailAgent" -ErrorAction SilentlyContinue) { $true } else { $false })
+    }
+    return $Service
+}
+
+function Assert-IsAgentServiceRegistered {
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session)
+
+    $Res = Test-IsAgentServiceRegistered -Session $Session
+    if (!$Res) {
```

Comment:
This `$Res` variable could have more meaningful name (or maybe just get rid of it)